### PR TITLE
Add php55 and php73 docker images for tests

### PR DESCRIPTION
**Pull request by tmatsuo**
cc @dwsupplee

### CODE REVIEW

**Comment by bshaffer** [comment]
This is great! Super great to be adding 7.3 so quickly! A few questions here:

1. Is there a reason we are adding PHP 5.5 when it's deprecated? I can see this being justified by the fact that `google/cloud` supports it, but this will add a lot of overhead (and will almost certainly require additional work to get the systems tests to pass). I would prefer to not add it if possible.

2. Is there a reason you aren't using the existing `Dockerfile.template` to keep these Dockerfiles consistent with each other?

**Comment by tmatsuo** [comment]
1. google/cloud supports it, so we need to have test with php 5.5.

2. Dockerfile.template depends on gcr.io/google-appengine/phpx there are no images for PHP 5.5 and 7.3

**Comment by tmatsuo** [comment]
You don't have to run the test against PHP 5.5. Only google-cloud-php will use that image.

**Comment by tmatsuo** [comment]
If you really want to maintain the docker images for 1) php-docs-samples and 2) google-cloud-php, that's fine, but considering the requirements will be very similar, I suggest we use the same images.

**Comment by bshaffer** [comment]
It doesn't really make sense to keep Dockerfiles here that are used in `google-cloud-php`. If `google-cloud-php` wants to use Dockerfiles in this repository, that's fine, but if they're only being used by `google-cloud-php`, then I think they belong in that repository.

Also, I believe we are moving the dockerfiles to a dedicated repository for container files anyway?

**Comment by tmatsuo** [comment]
> Also, I believe we are moving the dockerfiles to a dedicated repository for container files anyway?
Yes. Eventually the `.kokoro/docker` directory will be moved to the new repo. When moving, it is easier to have all the Dockerfiles in one place, so that's my motivation to add them here.

If you don't like this, feel free to close this PR. I'll add it to somewhere else.

File: .kokoro/docker/php55/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,98 @@
+FROM gcr.io/gcp-runtimes/ubuntu_16_0_4
```
Comment (bshaffer) [inline .kokoro/docker/php55/Dockerfile]: Just one small request - let's add a comment up here explaining this image is built for `GoogleCloudPlatform/google-cloud-php` testing.

Comment (tmatsuo) [reply .kokoro/docker/php55/Dockerfile]: Done

**Pull request merged**